### PR TITLE
Add interaction

### PR DESCRIPTION
**Pull request by lorenzoworx**
In this project, we are required to add some functionality to our application to make it interactive.  The user will also be able to mark a task as complete by selecting the corresponding checkbox (or undo it by unchecking the checkbox). The updated tasks list will be stored in *local storage*.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>To-do List</title>
```
Comment (ichala) [inline src/index.html:6]: - [ ] to keep your repo clean , kindly remove any commented code from your project 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- App works as it suppose to be ✔️ 


 ## Required Changes ♻️


- [x] **_🛑 To keep your readme professional ,Kindly fix the MIT LICENCE link and any broken link in your readme file 🙏_**
- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (lorenzoworx) [reply src/index.html]: Hey, thanks for the feedback. I've made the required changes. I would like to know how you are able to add the gif to the review markdown. Anytime I try it, it doesn't show up

**Review by wayungi — APPROVED** [review]
Hi @lorenzoworx , 
image

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (ichala) [reply src/index.html]: you can upload it normally as a file 🙏

**Pull request merged**